packages/gatsby/cache-dir/static-entry.js

```diff
@@ -5,13 +5,9 @@ import { kebabCase, get, merge, isArray, isString } from "lodash"
 import apiRunner from "./api-runner-ssr"
 import pages from "./pages.json"
 import syncRequires from "./sync-requires"
+import requireWithFallback from './require-with-fallback'
 
-let Html
-try {
-  Html = require(`../src/html`)
-} catch (e) {
-  Html = require(`./default-html`)
-}
+const Html = requireWithFallback(`../src/html`, `./default-html`)
```

Comment:
I don't think you can abstract this actually. Webpack statically analyzes requires, if the require path is completely dynamic the chances of stuff going wrong are high. like it including ALL files in the root in the bundle